dts/xtensa/intel/intel_cavs15.dtsi

```diff
@@ -123,8 +123,8 @@
 			interrupt-parent = <&core_intc>;
 		};
 
-		idc: idc@1200 {
-			compatible = "intel,cavs-idc";
+		intel_adsp_idc: idc@1200 {
```

Comment:
the intel in `intel_adsp_idc` seems redundant, the compatible already has the vendor, so just keep it `adsp_idc`.